dist/images/run-smart-nic-ovn.sh

```diff
@@ -0,0 +1,8 @@
+docker run --pid host --network host --user=0 --name ovn -dit --cap-add=SYS_NICE -v /var/run/secrets:/var/run/secrets \
```

Comment:
@moshe010 @adrianchiris What do you think about using [https://docs.docker.com/compose/](docker-composer)